pkg/controller/types/egress.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/vmware-tanzu/antrea/pkg/apis/controlplane"
+)
+
+// EgressGroup describes a set of GroupMembers to apply Egress to.
+type EgressGroup struct {
```

Comment:
It could share it with NetworkPolicy. I didn't do it to avoid touching NetworkPolicy stuff in this PR. If you prefer, I could move AppliedToGroup from networkpolicy.go to group.go in this PR and reuse it. You mean AppliedGroup or AppliedToGroup?

Reply:
I meant AppliedToGroup. If you want to reduce the change, we can refactor later - not much time left for this release.

Reply:
Do you think we can rename it to AppliedToGroup if later we want to unify with NetworkPolicy AppliedToGroup?

Reply:
Yes, it can. Should we do it when unifying with NetworkPolicy AppliedToGroup to avoid confusion that the API is named EgressGroup while its stored struct is AppliedToGroup?

Reply:
How about adding a comment here.

Reply:
done